code/unit_tests/atmospherics_tests.dm

```diff
@@ -254,7 +256,7 @@
 		var/list/after_gas_mixes = create_gas_mixes(gas_mix_data)
 
 		var/list/mix_sources = list()
-		var/list/all_gasses = subtypesof(/decl/material/gas)
+		var/list/all_gasses = decls_repository.get_decls_of_subtype(/decl/material/gas)
```

Comment:
I'd say this should probably be `get_decl_paths_of_subtype()` just so that it returns a list of paths like subtypesof, but I also can't tell what this code is doing at all; gasid isn't used, so it seems like it's just weighting it by the count of all gases that could exist? Is this sane?